File: Actividad 5/Problema 3/main.cpp
#include <iostream>
using namespace std;

class Fecha
{
    //Atributos
    public:
        int mes;
        int dia;
        int anio;

    //Constructor
    Fecha()
    {
        mes = 0;
        dia = 0;
        anio = 0;
    }
    
    //Metodos
    void mostrarFecha()
    {
        cout << mes << " / " << dia << " / " << anio << endl;
    }
};


int main()
{
    //FechaTest
    //Crear primer objeto
    Fecha fecha1;

    cout << "Ingresa el dia: ";
    cin >> fecha1.dia;

    cout << "Ingresa el mes (numero): ";
    cin >> fecha1.mes;

    cout << "Ingresa el año: ";
    cin >> fecha1.anio;
    
    fecha1.mostrarFecha();
    
    return 0;
}

File: Actividad 5/Problema 3/main.py
class Fecha:
    #Atributos
    def __init__(self):
        self.mes = 0
        self.dia = 0
        self.anio = 0 
        
    #Metodo
    def mostrarFecha(self):
        print(self.mes, " / ", self.dia, " / ", self.anio)

#Objeto
fecha1 = Fecha()

#FacturaTest
fecha1.mes = int(input("Ingresa el dia: "))
fecha1.dia = int(input("Ingresa el mes: "))
fecha1.anio = int(input("Ingresa el año: "))

fecha1.mostrarFecha()